### DISCUSSION TITLE
Modal.method()设置Token无效

### DISCUSSION THREAD

**Discussion by WT-Road** [ant-design/ant-design#40228] [Q&A] [answered]
Modal.info函数设置Token没有效果,是哪里有问题?
![image](https://user-images.githubusercontent.com/14371925/212346567-7a19befc-6e4d-4c47-9c9b-a3dc5f1a4d0d.png)

设置成功
![image](https://user-images.githubusercontent.com/14371925/212346932-2be14927-aaae-4b31-a7cf-c13fb3f4ebff.png)

**Comment by zombieJ** [comment, marked as answer]
静态方法不消费 ConfigProvider 数据，使用 hooks 版本即可： https://ant.design/components/modal-cn#components-modal-demo-hooks
